idl/github.com/uber/cadence/shared.thrift

```diff
@@ -54,6 +54,24 @@ exception QueryFailedError {
   1: required string message
 }
 
+enum WorkflowIdReusePolicy {
```

Comment:
Is it not merged master or why do you have these changes mixed up with the signal ones?

Reply:
This is just a sync from latest master